src/routes/docs/products/ai/integrations/perplexity/+page.markdoc

```diff
@@ -13,4 +13,310 @@ Integrating Perplexity into your Appwrite project is simple. This tutorial will
+        style="max-width: 50rem"
+      >
+        Use this page to test your implementation with Perplexity. Enter
+        text and receive the model output as a response.
+      </p>
+    </div>
+  </div>
+</main>
+```
+
+After this we're going to add the actual form that will allow us to make requests to our API, within the `<main>` tag and after the existing `<div>` add the following:
+
+```html
+<div class="container u-margin-block-start-negative-56"
+  x-data="{ prompt: '', answer: '', loading: false }">
+  <div class="card u-flex u-gap-24 u-flex-vertical">
+    <div class="u-flex u-cross-center u-gap-8">
+      <div
+        class="input-text-wrapper is-with-end-button u-width-full-line"
+      >
+        <input x-model="prompt" type="search" placeholder="Question" />
+        <div class="icon-search" aria-hidden="true"></div>
+      </div>
+
+      <button
+        class="button"
+        x-bind:disabled="loading"
+        x-on:click="async () => { loading = true; answer = ''; try { answer = await onSubmit(prompt) } catch(err) { console.error(err); } finally { loading = false; } }"
+      >
+        <span class="text">Submit</span>
+      </button>
+    </div>
+    <template x-if="answer">
+      <div class="u-flex u-flex-vertical u-gap-12">
+        <div class="u-flex u-flex-vertical u-gap-12 card">
+          <div class="u-flex u-gap-12">
+            <h5 class="eyebrow-heading-2">Perplexity:</h5>
+          </div>
+
+          <div style="overflow-x: hidden; line-break: anywhere">
+            <p class="u-color-text-gray" x-text="answer"></p>
+          </div>
+        </div>
+      </div>
+    </template>
+  </div>
+</div>
+```
+
+This code might seem quite complex if you've never used AlpineJS so we'll break it down a bit:
+
+1. `x-data="{ prompt: '', answer: '', loading: false }"` defines our state for the application.
+2. `x-model="prompt"` is a two-way binding that will bind the value of the input to the `prompt` variable which will update our state for us.
+3. `x-bind:disabled="loading"` will disable the button if the `loading` state variable is true.
+4. The `x-on:click` binds an event listener that will call the `onSubmit` function we defined earlier within `<script>` when the button is clicked. This function will set the `loading` state to true, clear the `answer` variable, and then call the `onSubmit` function. If the function fails, it will log the error to the console and set the `loading` state back to false.
+5. `x-if="answer"` will only render the completion if the `answer` variable is not empty.
+6. Finally, `x-text="answer"` will bind the text of the completion to the `answer` variable.
+
+Putting all this together gives us a nice form that will allow us to send prompts to our API and display the completion.
+{% /section %}
+
+{% section #step-8 step=8 title="Deploy the function" %}
+With the function complete, we can now deploy it to Appwrite.
+{% partial file="deploy-function.md" /%}
```

Comment:
Stick with deploying with Git. You use an Appwrite function template, makes no sense to use the other options.